apl/aggregation-function/percentileif.mdx

```diff
@@ -0,0 +1,152 @@
+
+You can use `percentileif` to gain insights in various scenarios, such as:
+
+- Identifying response time percentiles for HTTP requests from specific regions.
+- Calculating percentiles of span durations for specific service types in OpenTelemetry traces.
+- Analyzing security events by percentile within defined risk categories.
+
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+The `percentileif` aggregation in APL works similarly to `percentile` combined with conditional filtering in SPL. However, APL integrates the condition directly into the aggregation for simplicity.
+
+<CodeGroup>
+```sql Splunk example
+stats perc95(req_duration_ms) as p95 where geo.country="US"
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| summarize percentileif(req_duration_ms, 95, geo.country == 'US')
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+In SQL, you typically calculate percentiles using window functions or aggregate functions combined with a `WHERE` clause. APL simplifies this by embedding the condition directly in the `percentileif` aggregation.
+
+<CodeGroup>
+```sql SQL example
+SELECT PERCENTILE_CONT(0.95) WITHIN GROUP (ORDER BY req_duration_ms)
+FROM sample_http_logs
+WHERE geo_country = 'US'
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| summarize percentileif(req_duration_ms, 95, geo.country == 'US')
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
+
+```kusto
+summarize percentileif(NumericColumn, Percentile, Predicate)
+```
+
+### Parameters
+
+| Parameter       | Description                                                                                                  |
+|------------------|--------------------------------------------------------------------------------------------------------------|
+| `NumericColumn` | The numeric column from which to calculate the percentile.                                                   |
+| `Percentile`    | A number between 0 and 100 that specifies the percentile to calculate.                                        |
+| `Predicate`     | A boolean expression that filters rows to include in the calculation.                                         |
+
+### Returns
+
+The function returns a single numeric value representing the specified percentile of the `NumericColumn` for rows where the `Predicate` evaluates to `true`.
+
+## Use case examples
+
+<Tabs>
+<Tab title="Log analysis">
+
+You can use `percentileif` to analyze request durations for specific HTTP methods.
+
+**Query**
+
+```kusto
+['sample-http-logs']
+| summarize percentileif(req_duration_ms, 90, method == 'POST')
```

Comment:
For one of these I think it would be good to do two, just so people can see it's working. So perhaps:

```
| summarize post_p90 = percentileif(req_duration_ms, 90, method == "POST"), 
            get_p90 = percentileif(req_duration_ms, 90, method == "GET")
         by bin_auto(_time)
```